shared/actions/chat/inbox.js

```diff
@@ -238,7 +241,7 @@ function* _chatInboxConversationSubSaga({conv}) {
   // Wait for an idle
   yield call(onIdlePromise, 100)
   // TODO might be better to make this a put with an associated takeEvery
-  yield fork(processConversation, conv)
+  yield spawn(processConversation, conv)
```

Comment:
I think we have the data, we should finish the processConversation, right? I mean that's what the old code effectively did